test/testinput/prepbufr_adpsfc_api.py

```diff
@@ -53,7 +52,7 @@ def test_bufr_to_ioda():
    print("Get time")
    dhr = r.get('obsTimeMinusCycleTime') #Needs to be converted to seconds since Epoch time from [-3,3]
    print("cycleTimeSinceEpoch") #For now, file time is put in manually 
-   cycleTimeSinceEpoch = np.int64(calendar.timegm(time.strptime('2021 08 01 00 00', '%Y %m %d %H %M')))
+   cycleTimeSinceEpoch = np.int64(calendar.timegm(time.strptime(date, '%Y%m%d%H%M')))
```

Comment:
Don't need the spaces in the format?
```suggestion
   cycleTimeSinceEpoch = np.int64(calendar.timegm(time.strptime(date, '%Y %m %d %H %M')))
```